### DISCUSSION TITLE
[Question][Hw5] Warning: Unable to read directory node_modules

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#623] [Q&A]
I am receiving the following warning when I run `make`:

```
Warning: Unable to read directory node_modules. Ignoring.
Remove this message by ignoring by adding:
(dirs \ node_modules)
to the dune file: dune
Reason: Permission denied
```
Could you help me?

**Comment by RiceBiscuits** [comment]
If you are using multi accounts(e.g., root and other account) on your linux, please try `sudo`.

**Comment by mrpositron** [comment]
I did not need it when I was doing previous assignments. I am not sure why it is happening now.

**Comment by mrpositron** [comment]
Just cloned the repo again. It solved the problem.
